volatility3/framework/plugins/isfinfo.py

```diff
@@ -96,16 +97,12 @@ def _generator(self):
                     if filter_item in isf_file:
                         filtered_list.append(isf_file)
 
-        try:
-            import jsonschema
-
-            if not self.config["validate"]:
```

Comment:
This is actually an important bit of code, not sure why it took that out there...

Reply:
Oh, I see, it moved it up into the if statement

Reply:
its not taken out, because the import isn't used it uses  `find_spec` from `importlib.util`

Reply:
Yeah, it was the if statement I was objecting to losing, but actually it moved it into the if statement and it all works out the same.  5:)